README.md

```diff
@@ -53,7 +56,7 @@ You are going to need a few things to begin. Firstly, NodeJS. There are a few wa
 - [Official installer](https://nodejs.org/en/)
 - [Node Version Manager](https://github.com/nvm-sh/nvm)
 - [Homebrew](https://formulae.brew.sh/formula/node)
-
+$$
```

Comment:
A little bit confused about the inclusion of this `$$` (perhaps I'm missing something / don't understand it's use) - could you clarify what this provides to the markdown please?